pkg/broker/interfaces.go

```diff
@@ -0,0 +1,10 @@
+package broker
+
+import "github.com/openshift/ansible-service-broker/pkg/apb"
+
+// JobStateDAO defines the actions that can be preformed on the JobState resource
+type JobStateDAO interface {
+	GetState(instanceUUID, operation string) (apb.JobState, error)
+	SetState(id string, state apb.JobState) error
+	GetSvcInstJobsByState(instanceID string, reqState apb.State) ([]apb.JobState, error)
+}
```

Comment:
I'd like to map out what the interfaces for the dao will be, like I mentioned earlier there could be many which could blow up some of the methods. I think the interfaces are the only solution just need to be smart in how we define them.